test/integration/connect/test_connect.py

```diff
@@ -297,6 +297,57 @@ def test_connect_and_get_commands_errors(tt_cmd, tmpdir_with_cfg):
     finally:
         stop_app(tt_cmd, tmpdir, "test_app")
 
+def test_connect_and_handle_lua_parse_error(tt_cmd, tmpdir_with_cfg):
+    tmpdir = tmpdir_with_cfg
+    empty_file = "empty.lua"
+    # The test application file.
+    test_app_path = os.path.join(os.path.dirname(__file__), "test_localhost_app", "test_app.lua")
+    # The test file.
+    empty_file_path = os.path.join(os.path.dirname(__file__), "test_file", empty_file)
+    # Copy test data into temporary directory.
+    copy_data(tmpdir, [test_app_path, empty_file_path])
+
+    # Start an instance.
+    start_app(tt_cmd, tmpdir, "test_app")
+
+    # Check for start.
+    file = wait_file(os.path.join(tmpdir, 'test_app'), 'ready', [])
+    assert file != ""
+
+    cmd="hello)"
+
+    formats = {}
+    formats["yaml"] = f"""---
+- error: '[string "{cmd}"]:1: ''='' expected near '')'''
+...
+
+"""
+
+    formats["lua"] = f'''{{error = "[string "{cmd}"]:1: '=' expected near ')'"}};\n'''
+
+    formats["table"] = (
+        '''+--------------------------------------------+\n'''
+        '''| error                                      |\n'''
+        '''+--------------------------------------------+\n'''
+        f'''| [string "{cmd}"]:1: '=' expected near ')' |\n'''
+        '''+--------------------------------------------+\n'''
+    )
+
+    formats["ttable"] = (
+        '''+-------+--------------------------------------------+\n'''
+        f'''| error | [string "{cmd}"]:1: '=' expected near ')' |\n'''
+        '''+-------+--------------------------------------------+\n'''
+    )
+
+    try:
+        for format, expected in formats.items():
+            ret, output = try_execute_on_instance(tt_cmd, tmpdir, "localhost:3013", stdin=cmd,  opts={"-x": format})
```

Comment:
If you run [python linter](https://github.com/tarantool/tt/blob/da304aae43ed37ab80dd1c397dada1eddbc21797/.github/actions/static-code-check/action.yml#L33) , you will see than this line is too long.

```suggestion
            ret, output = try_execute_on_instance(tt_cmd, tmpdir, "localhost:3013",
                                                  stdin=cmd,  opts={"-x": format})
```

Reply:
My bad.
I'm not a python programmer and accidentally disabled python linter with heavy golangci)